Src/HALAL/Services/Communication/SPI/SPI.cpp

```diff
@@ -110,3 +118,47 @@ void HAL_SPI_ErrorCallback(SPI_HandleTypeDef *hspi){
     //TODO: Fault, SPI error
     return;
 }
+
+void SPI::init(SPI::Instance* spi){
+	if (spi->initialized) {
+		return;
+	}
+
+	spi->hspi->Instance = spi->instance;
```

Comment:
```suggestion

	spi->hspi->Instance = spi->instance;
	
	InitType& init_data  = spi->hspi->Init
```
podrias hacer esta referencia para la funcion, quedarian todas las asignaciones init_data.Mode en vez de spi->hspi->Init, mucho mas limpio

Reply:
vale, ahora lo cambio.